test/mongodb.test.js

```diff
@@ -2799,6 +2799,21 @@ describe('mongodb connector', function() {
     });
   });
 
+  it('should support count without where', function(done) {
+    const POST_NUMBER = 35;
+    const posts = [];
+    for (let i = 0; i < POST_NUMBER; i++)
```

Comment:
nitpick: let's include the curly braces for this `for` loop, even though it's one line.

Reply:
Done